### DISCUSSION TITLE
Landingpages

### DISCUSSION THREAD

**Discussion by fradarr** [3Stones-io/omedis#238] [Ideas]
1) Before an Organisation is created it will be a "pech" landing page, polished and aiming to sell the SAAS product we are creating

2) As soon as an Organisation (tenant) is created the landing page for this Organisation should be 
a) a login screen if not loged in
b) dashboard based on the role / group

**Comment by almirsarajcic** [comment]
Let's simplify things by having a shared vocabulary.
I suggest we call the 1st page the landing page and the second one the onboarding page so we don't get confused by calling both of them the landing page.

Who's gonna design that?

**Comment by fradarr** [comment]
Dashboard accepted!
